apache-rat-core/src/main/java/org/apache/rat/configuration/builders/SpdxBuilder.java

```diff
@@ -32,22 +31,22 @@ public class SpdxBuilder extends AbstractBuilder {
 
     /**
      * sets the name for the SPDX matcher
-     * @param name
-     * @return
+     * @param name The text that follows the colon ':' in the SPDX tag.
+     * @return this builder for chaining.
      */
     public SpdxBuilder setName(String name) {
-        Objects.requireNonNull(name, "name must not be null");
+        Objects.requireNonNull(name, "spdx name must not be null");
```

Comment:
Does SPDX require uppercasing, if so we could enforce it here and in the constructor?